src/components/AddBook.js

```diff
@@ -1,6 +1,6 @@
 import { useSelector, useDispatch } from 'react-redux';
 import { useState } from 'react';
-import { addBook } from '../redux/books/booksSlice';
+import { createBook } from '../redux/books/booksSlice';
 
 const AddBook = () => {
```

Comment:
- [ ] Well done @BenMKT on achieving this far with the project requirements 👏. However, there seems to be inconsistency in creating books, where sometimes the book is created successfully but other times the title and author are empty. 

#### This :point_down:  is a recording of the issue.
[Screencast from 25.08.2023 20:40:06.webm](https://github.com/BenMKT/Bookstore/assets/89228743/80ffb330-3e4a-4eef-a77a-26ce397d4cf9)


I spent some time looking over your code to figure out what might have been the source of the problem and I discovered that you are not giving the **title** and the **author** as **values** to the **inputs**. So that is the issue that arises when writing books. 😃 

Please remember that it is important to connect input **values** to state variables using the **value** attribute. This creates a direct link between what is shown on the screen and what is saved in our application's memory. When we modify the state variable, such as typing in an input field, the displayed value will update automatically to match what we typed. 🤓 

To fix the issue please ensure to update the input in the addBook component like follow: 

```
        <input
          type="text"
          value={title}
          onChange={(e) => {
            setTitle(e.target.value);
          }}
          placeholder="Book Title"
        />
        <input
          type="text"
          value={author}
          onChange={(e) => {
            setAuthor(e.target.value);
          }}
          placeholder="Author"
        />
```
You can notice I've added the **value** attribute to the **input elements**, making them responsive to changes in the state. This ensures that the input fields always show the current content from our state, preventing any mismatch between what we're typing and what's displayed. 👍️ 😊

Reply:
- Added `value` attributes to the `input` elements making them responsive to changes in the state to fix bug.